### ISSUE TITLE
full project sample for cross-platform development using g3m as a lib?

### ISSUE THREAD

**Issue by akosmaroy** [glob3mobile/g3m#52]
it would be nice to have a full sample project, that demonstrates how one would develop a cross-platform app that references g3m as a library? this would entail a C++ project that depends on the C++ code of G3M, and then packaging this into iOS, Android and GWT project, using cross-compilation for Android and GWT. this would be similar to the Demo apps, with the organization difference that G3M would be an external library to the project.

here the crucial thing is how the cross-compile the code, and organize the code. suppose one would purchase a license for the cross-compiler used, how would one set this up?

(I tried and failed to even reproduce the cross-compilation done for the project itself. didn't have a chance to try a project that would reference G3M as an external library)

**Comment by vidaltoboso** [comment]
Hi Akos,
I don't know if I respond to your question with what I commented below: 
We have created C++ static libraries it depends on the g3m project (G3MiOSSDK). The conversion process is similar, but the "Code Converter" can't work directly with "libG3MiOSSDK.a" or "g3msharedsdk.jar", so to work correctly, you must add the "Commons" directory, of G3MiOSSDK, to "Converter Options -> Header Files" (.dat). 
If these libraries are written with C++ namespaces, as you suggested in issue https://github.com/glob3mobile/g3m/issues/34, the "Code Converter" generates the "imports", "packages" and directories structure of java directly, and file ".bat" is simplest.
